apps/web/components/AddressDisplay/AddressDisplaySkeleton.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <p class="h-4 w-36 bg-gray-300 rounded animate-pulse" />
+  <p class="mt-2 h-4 w-36 bg-gray-300 rounded animate-pulse" />
+  <p class="mt-2 h-4 w-36 bg-gray-300 rounded animate-pulse" />
+  <p class="mt-2 h-4 w-36 bg-gray-300 rounded animate-pulse" />
+  <p class="mt-2 h-4 w-36 bg-gray-300 rounded animate-pulse" />
+</template>
```

Comment:
```suggestion
<template>
  <div class="flex gap-2">
    <span class="h-3 w-20 bg-gray-300 rounded animate-pulse" />
    <span class="h-3 w-20 bg-gray-300 rounded animate-pulse" />
  </div>
  <div class="flex gap-2 mt-2">
    <span class="h-3 w-24 bg-gray-300 rounded animate-pulse" />
    <span class="h-3 w-5 bg-gray-300 rounded animate-pulse" />
  </div>
  <div class="flex gap-2 mt-2">
    <span class="h-3 w-12 bg-gray-300 rounded animate-pulse" />
    <span class="h-3 w-12 bg-gray-300 rounded animate-pulse" />
  </div>
  <p class="mt-2 h-3 w-20 bg-gray-300 rounded animate-pulse" />
</template>
```

more like this, needs to br adjusted in height and margins so that there is little to non shift when the data loads